README.md

```diff
@@ -108,6 +113,33 @@ After updating the `otelcol-config.yml` start the demo by running
 `docker compose up`. After a while you should see the traces flowing into
 your backend as well.
 
+### Kubernetes
+
+We provide a [OpenTelemetry
+Demo](https://github.com/open-telemetry/opentelemetry-demo) helm chart to
+quickly get a kubernetes cluster running.
```

Comment:
I think this could use some wordsmitthing.  The helm chart doesn't help get a k8s cluster running, it depends on a cluster already existing and running.  The helm chart helps "deploy" the opentelemetry-demo to the cluster.

Reply:
Ahh I meant to use the helm repo link but forgot to update. Changed the following wording too. Thanks for the suggestions!